src/app/qgsdatasourcemanagerdialog.h

```diff
@@ -79,6 +81,10 @@ class QgsDataSourceManagerDialog : public QgsOptionsDialogBase
     void handleDropUriList( const QgsMimeDataUtils::UriList & );
     //! Update project home directory
     void updateProjectHome();
+    //! Connections changed
+    void connectionsChanged( );
+    // internal signal
+    void dlg_refresh( );
```

Comment:
Ideally this should not be handled by a signal, but rather through a common base class for dialogs - adding more signals like this will make the code difficult to read / understand. If starting a base class for the individual provider dialogs is too much of the work, I would suggest to at least rename the signal (e.g. providerDialogsRefreshRequested) and document it...